package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "not-valid",
-  "version": "2.0.0",
```

Comment:
What's actually changed?  I can't see anything in package-lock.json that's changed either?